src/e2e.js

```diff
@@ -78,6 +111,7 @@ mye2e.prototype.exportKey = function() {
 };
 
 mye2e.prototype.signEncrypt = function(data, encryptKey, sign) {
+  Promise.resolve(refreshBuffer(5000));
```

Comment:
does that also reduce the flakyness?

On Tue, Jul 14, 2015 at 4:49 PM, soycode <notifications@github.com> wrote:

> In src/e2e.js
> <https://github.com/freedomjs/freedom-pgp-e2e/pull/38#discussion_r34633324>
> :
>
> > @@ -78,6 +111,7 @@ mye2e.prototype.exportKey = function() {
> >  };
> >
> >  mye2e.prototype.signEncrypt = function(data, encryptKey, sign) {
> > +  Promise.resolve(refreshBuffer(5000));
>
> Yeah that would be better, this was just a lazy first attack that got
> things running. Adding the .then at the bottom as in the setup changed
> the return type, but it should be changeable in another way.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/freedomjs/freedom-pgp-e2e/pull/38/files#r34633324>.
>